roles/ui/files/FWO.UI/Shared/ConnectionTable.razor

```diff
@@ -59,17 +66,22 @@
     {
         <Column TableItem="ModellingConnection" Title="@(userConfig.GetText("owner"))" Field="@(x => x.App.Name)" Sortable="true" Filterable="true">
             <Template>
-                @(context.App?.Display(userConfig.GetText("common_service")))
```

Comment:
let me re-phrase this: can you create two methods which display the name and the extappid depending on the common service parameter (meaning the common service string should be added to the name (as we do currenlty)